docs/configuration.md

```diff
@@ -16,44 +16,64 @@ For example, by default Trivy displays vulnerabilities with all severity levels
 in the `starboard` ConfigMap:
 
 ```
-kubectl patch configmap starboard -n starboard \
+kubectl patch cm starboard -n <starboard_operator> \
   --type merge \
-  -p '{"data": {"trivy.severity":"HIGH,CRITICAL"}}'
+  -p "$(cat <<EOF
+{
+  "data": {
+    "trivy.severity": "HIGH,CRITICAL"
+  }
+}
+EOF
+)"
 ```
 
 To set the GitHub token used by Trivy in `Standalone` mode add the
 `trivy.githubToken` value to the `starboard` secret instead:
 
 ```
 GITHUB_TOKEN=<your token>
-kubectl patch secret starboard -n starboard \
+
+kubectl patch secret starboard -n <starboard_operator> \
   --type merge \
-  -p "{\"data\": {\"trivy.githubToken\":\"$(echo -n $GITHUB_TOKEN | base64)\"}}"
+  -p "$(cat <<EOF
+{
+  "data": {
+    "trivy.githubToken": "$(echo -n $GITHUB_TOKEN | base64)"
+  }
+}
+EOF
+)"
 ```
 
 The following tables list available configuration parameters with their default
 values.
 
-| CONFIGMAP KEY             | DEFAULT                                                | DESCRIPTION |
-| ------------------------- | ------------------------------------------------------ | ----------- |
-| `trivy.httpProxy`         | N/A                                                    | The HTTP proxy used by Trivy to download the vulnerabilities database from GitHub. Only applicable in `Standalone` mode. |
-| `trivy.severity`          | `UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL`                     | A comma separated list of severity levels reported by Trivy |
-| `trivy.imageRef`          | `docker.io/aquasec/trivy:0.14.0`                       | Trivy image reference |
-| `trivy.mode`              | `Standalone`                                           | Trivy client mode. Either `Standalone` or `ClientServer`. Depending on the active mode other settings might be applicable or required. |
-| `trivy.serverURL`         | `http://trivy-server.trivy-server:4954`                | The endpoint URL of the Trivy server. Required in `ClientServer` mode. |
-| `trivy.serverTokenHeader` | `Trivy-Token`                                          | The name of the HTTP header to send the authentication token to Trivy server. Only application in `ClientServer` mode when `trivy.serverToken` is specified. |
-| `polaris.config.yaml`     | [Check the default value here][default-polaris-config] | Polaris configuration file |
+| CONFIGMAP KEY                  | DEFAULT                                                | DESCRIPTION |
```

Comment:
I wonder if we should explicitly say somewhere that you only need to configure the settings for the scanner you're using (i.e. trivy.* is only used if scanner is set to Trivy)

Reply:
Maybe also a reference to the Integrations section where it's all laid out clearly what you need to do with these settings

Reply:
Good point. I added both.